README.md

```diff
@@ -1,12 +1,35 @@
 # Wynntils Data-Storage
 
-## worldmap
+## Wynntils maps
+
+Generating maps for Wynntils (both Artemis and Legacy) is a two-step process.
+First, we must gather "tiles" of maps using Journeymap, and then we can process
+these tiles into the png files and additional metadata needed by Wynntils.
+
+It is important to keep a high quality of the tile data. This will allow us to
+make minor updates to the map by just gathering the modified tiles, and then
+easily update the map using a simple script.
+
+### The worldmap directory
+
 The `worldmap` directory contains all data needed to re-generate the in-game
 worldmap/minimap.
 
+`worldmap/journeymap-data` contains tiles that are created by Journeymap, and
+special care must be taken to keep them correctly updated. Make sure to read the
+following instructions carefully!
+
+`worldmap/masks` contains black and white masks that determine which part of the
+mapped areas that should be included, and which should be transparent. See below
+on adding new areas on how to update these. The single mask used by Legacy is
+instead stored in `reference/map-mask.png`.
+
 The script `scripts/update-journeymap.sh` will help you with updating the
 worldmap, to make sure this is done as smoothly as possible for you, and with
 repeatable results.
+
+## Gathering tiles data
```

Comment:
Missing part

Reply:
Actually, this is a level 2 heading. The level 3 headings below all is part of how to gather tiles data. (Basically the readme prior to this PR was just about gathering tiles data.) But maybe this is not entirely clear.